src/routes.jsx

```diff
@@ -0,0 +1,36 @@
+
+// LIBRARIES
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+// Components
+import Navbar from './components/Navbar/Navbar';
+// import Calculator from './components/Calculator';
```

Comment:
- [ ] It will be nice to remove commented code from your code base, as this makes your code base dirty and less readable.